### DISCUSSION TITLE
API concepts

### DISCUSSION THREAD

**Discussion by NikithaPut** [tewhatuora/api-standards#10] [Q&A] [answered]
### Summary

1. Standards  Component Definitions :Do we need to add soap services as well under API providers/Consumers as we will be supporting soap services

### Which area of the standards does this apply to?

- [X] Part A - API Concepts
- [ ] Part B - API Security
- [ ] Part C - API Design and Development
- [ ] Part D - FHIR
- [ ] Community guidelines

**Comment by swithinfoote** [comment, marked as answer]
Hi @NikithaPut, see https://dhzf2wyhwgplf.cloudfront.net/api-development/API%20Design#types-of-api for details on SOAP support. It has been listed as `Deprecated` which means it is supported however building new SOAP interfaces should be discouraged.
Cheers.
